src/widgets/histogram/histogram-title-view.js

```diff
@@ -27,7 +27,7 @@ module.exports = cdb.core.View.extend({
     this.$el.html(
       template({
         title: this.widgetModel.get('title'),
-        isAutoStyleEnabled: this.widgetModel.isAutoStyleEnabled(),
+        isAutoStyleEnabled: this.widgetModel.isAutoStyleEnabled() && this.dataviewModel.layer.get('visible'),
```

Comment:
If widgetModel has a reference to the (dataviewModel) layer, I'll add this check in the `isAutoStyleEnabled` method.